### PR TITLE
User's location on the map

### PR DESCRIPTION
**Pull request by Karyum**
we made it so when the page loads it would put a marker on the location of the user and error handled  it

related to #26 #23 #12

### CODE REVIEW

File: app/public/map.js [hunk 1]
```diff
@@ -5,12 +5,29 @@
...
+
+  mymap.locate({setView: true, maxZoom: 20})
+
+  function onLocationFound (e) {
+    var radius = e.accuracy / 2
+
+    L.marker(e.latlng).addTo(mymap)
+
+    L.circle(e.latlng, radius).addTo(mymap)
+  }
```
Comment (mattlub) [inline app/public/map.js]: In my opinion, we should be watching the location of the user.
Then onLocationFound would have to move the marker (or remove the previous marker and make a new one).

thoughts?

Comment (mattlub) [reply app/public/map.js]: Also, I think we should display a marker more like this:
http://www.objectivehtml.com/assets/media/current-location.png

File: app/public/map.js [hunk 2]
```diff
@@ -5,12 +5,29 @@
...
+    minZoom: 13
   })
 
   L.tileLayer('https://api.mapbox.com/styles/v1/mapbox/streets-v10/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1Ijoic3V1dXVoYSIsImEiOiJjajI2N3QwZm0wMDE1MnFwb3NnYnhwaG55In0.sZgdmc9B4GG9X4Bx5o3NWg', {
     attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="http://mapbox.com">Mapbox</a>',
     id: 'mapbox.light',
     accessToken: 'pk.eyJ1Ijoia2FyeXVtIiwiYSI6ImNqMjAzNGU4ZjAxa3EycW4xazFxcHZ6a2QifQ.m_dNO1l1sMkM7r4d5nlRRQ'
   }).addTo(mymap)
+
+  mymap.locate({setView: true, maxZoom: 20})
```
Comment (mattlub) [inline app/public/map.js:17]: why max zoom?

Comment (Karyum) [reply app/public/map.js:17]: this zooms in on the user

File: app/public/map.js (hunk 1 above)
Comment (Karyum) [reply app/public/map.js]: yeah ill work on the watching user location and ill see what i can do with the icon

Comment (mattlub) [reply app/public/map.js]: after discussing yesterday, I think we don't need to implement the watch yet (what you've done does what the issue required)

**Review by mattlub — APPROVED** [review]

**Review by Suuuuha — APPROVED** [review]

**Pull request merged**